stylesheets/2048.css

```diff
@@ -1,8 +1,43 @@
 html {
   font: normal normal 30px/1 "Clear Sans", "Helvetica Neue", Arial, sans-serif;
+  font-family: 'Orbitron', sans-serif;
+}
+
+body {
+  margin: 0% 10%;
+}
+
+h1 {
+  font-size: 4rem;
+  text-align: center;
+  margin: 5px;
+  color: #555250;
+}
+
+p {
+  padding-top: 7px;
+  margin-bottom: 5px;
+  /*text-align: center;*/
+}
+
+img {
+  /*float: left;*/
+  position: relative;
+}
+
+footer {
+  text-align: center;
+}
+
+#dashboard {
+  /*margin: 0% 20%;*/
+  /*i don't know how to center this stuff arggh!!*/
```

Comment:
Yeah, centering is surprisingly difficult in CSS. If you can get by with `text-align`, as below, that's nice. For more involved situations, or centering visually, checkout CSS Flexbox. It's relatively new and buggy, and takes a while to wrap your brain around, but it's mostly usable these days. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Using_CSS_flexible_boxes